### PR TITLE
added storybook build npm script

### LINKED ISSUE #86: Ensure Storybook Deployment on Heroku/Netifly - develop branch deployment
## User story

Who: As a user

What: I want to be able to see Storybook Online like I see it locally installed locally.

Why: so that I can demo all the stories without having the project 

## Acceptance criteria

Make sure we can deploy storybook stories.

You can try to check this tutorial https://www.learnstorybook.com/intro-to-storybook/react/en/deploy/ and this https://storybook.js.org/docs/react/workflows/publish-storybook

## Implementation details

You maybe need to add this

image

### PR DESCRIPTION
**Pull request by ember-b-Moss**
# Description

Set up deployment for storybook.

Fixes #86 

# How to test?
[Visit deployed storybook](https://hyf-rate-my-cv-storybook.netlify.app/) 

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Comment by pdsorensen** [comment]
@ember-b-Moss does this gets triggered automatically or manually?

For reference, here are the docs: https://storybook.js.org/docs/react/api/cli-options

**Comment by ember-b-Moss** [comment]
> https://storybook.js.org/docs/react/api/cli-options

what do you mean by this? Production branch is develop. Everything that is on develop will be automatically online

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "db:clean": "cd ./src/server/ && knex migrate:rollback",
     "db:setup": "cd ./src/server/ && knex migrate:latest && knex seed:run",
     "storybook": "cd ./src/client/ && start-storybook -p 3007",
+    "build-storybook": "build-storybook --no-dll -s public -c ./src/client/.storybook --quiet",
```
Comment (ember-b-Moss) [inline package.json:19]: @pdsorensen this was the script i used?

**Review by pdsorensen — APPROVED** [review]
Looks good. The docs i posted, is just for general reference in case somebody wonders what the arguments means. 

Cheers,

**Pull request merged**